octopusdeploy_framework/schemas/space.go

```diff
@@ -100,7 +100,7 @@ func GetSpaceDatasourceSchema() map[string]datasourceSchema.Attribute {
 			Validators: []validator.String{
 				stringvalidator.LengthBetween(1, 20),
 			},
-			Required: true,
```

Comment:
This is part of the individual space data source I think this should stay required.

Reply:
Yes, this was a mistake